### PR TITLE
Nuget changes for internal published Nugets

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Nuget changes for internally published Nugets, some of these are used to be manually published

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

Nuget changes for internal published Nugets

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec [hunk 1]
```diff
@@ -15,8 +15,11 @@
             <group targetFramework=".NETFramework4.8" />
         </dependencies>
     </metadata>
+    <!--for now we assume that that the working directory root is dynamo/bin/AnyCPU/Release-->
     <files>
         <file src="**" target="lib\net48" />
+        <!--NET60 binaries, the path may change after we stop building NET48 solution-->
+        <file src="..\..\..\bin\NET60_Windows\Release\**" target="lib\net6.0" />
```
Comment (QilongTang) [inline tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec:22]: @sm6srw I remember the decision was to put the NET60 build as a different Nuget package right? Was there a different nuspec file?

File: tools/NuGet/template-artifactory/DynamoVisualProgramming.CoreRuntimeNet6.nuspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>DynamoVisualProgramming.CoreRuntimeNet6</id>
```
Comment (QilongTang) [inline tools/NuGet/template-artifactory/DynamoVisualProgramming.CoreRuntimeNet6.nuspec:4]: @sm6srw any reason for the new naming? I was thinking of keeping the name and just append framework name to be `DynamoCoreRuntimeNET60`

Comment (sm6srw) [reply tools/NuGet/template-artifactory/DynamoVisualProgramming.CoreRuntimeNet6.nuspec:4]: Not really. I am usually really bad at picking new names as you can see. I like your reasoning about the name better so please go ahead and rename it. We need to ping D4R and C3D about the change. Better to do it know than later.

**Review by sm6srw — APPROVED** [review]
I have disabled the temporary daily build that we had. I will remove that job completely when we know that this works.

File: tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntimeNET60.nuspec [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Assemblies required to start a DynamoModel and execute DesignScript code bundled along with their dependencies.
+            Built targeting Windows Net6 with exclusion of test libraries and symbol files. </description>
+        <copyright>Copyright Autodesk 2023</copyright>
+        <dependencies>
+            <group targetFramework="net6.0">
+            </group>
+        </dependencies>
+    </metadata>
+    <!--for now we assume that that the working directory root is dynamo/bin/AnyCPU/Release-->
```
Comment (aparajit-pratap) [inline tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntimeNET60.nuspec:20]: This should be `Dynamo\bin\NET60_Windows\Release`

Comment (QilongTang) [reply tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntimeNET60.nuspec:20]: No @aparajit-pratap See below, when you define the files, you are actually navigating from the working dir

**Pull request merged**